cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -419,8 +443,10 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
 
     prepareAndTestQuery(
       title = "Drop a constraint",
-      text = "A constraint can be dropped using the name with the `DROP CONSTRAINT constraint_name` command. " +
-        "It is the same command for unique property, property existence and node key constraints.",
+      text =
+        """A constraint can be dropped using the name with the `DROP CONSTRAINT constraint_name` command.
+          |It is the same command for unique property, property existence and node key constraints.
+          |The name of the constraint can be found using <<administration-constraints-list-constraint, `SHOW CONSTRAINTS`>>, given in the output column `name`.""".stripMargin,
```

Comment:
It is not easy to understand that `SHOW CONSTRAINTS` is a link here given that it is back-ticked, but maybe that is different in the new design? One option is to change it to 

the <<administration-constraints-list-constraint, `SHOW CONSTRAINT` command>> 

to make it consistent with the `DROP CONSTRAINT` link in the `SHOW CONSTRAINT` example .